### DISCUSSION TITLE
Please add video example of what this does.

### DISCUSSION THREAD

**Discussion by CuteWaterBeary** [Xaymar/obs-stream-effects#838] [General]
### Explain the Feature/Enhancement

Add a video showcasing the features of this plugin.

### What makes this feature/enhancement useful for StreamFX and its users?

Potential users are going to click off and not install because this has no examples and what if it looks terrible after spending a long time installing? People click off youtube videos after 20 seconds and the install here would definately take more than 20 seconds.

### Checklist

- [ ] I am offering a monetary bounty/reward for the implementation of this Feature/Enhancement.
- [ ] There is no other way to currently do this in StreamFX or OBS Studio.

### Any additional Information we need to know?

Please add some recorded examples so I can consider installing this.

**Comment by Xaymar** [comment]
> Potential users are going to click off and not install because this has no examples

If someone needs a video to understand what software does and what you can do with it, most likely they aren't in the target demographic of the software in the first place. 

> what if it looks terrible after spending a long time installing?

Installation of the plugin is a matter of seconds, or minutes on slower connections. Also the plugin itself does nothing unless instructed to, so you still need to learn how to create what you want to see anyway. This is the same as it is with any other effects.

> People click off youtube videos after 20 seconds and the install here would definately take more than 20 seconds.

Further proof that those users are not part of the target demographic.

> Please add some recorded examples so I can consider installing this.

How about just searching for StreamFX on the common platforms? https://www.youtube.com/results?search_query=streamfx

**Comment by CuteWaterBeary** [comment]
Im not gunna search for hundreds of plugins for all the different programs I use on youtube. Whats so hard about putting a link to that in your readme?
